articles/cloud-storage/cs-how-use-file-browsers.md

```diff
@@ -146,78 +146,64 @@ To install and configure S3 Browser:
     > [!NOTE]
     > Not all S3 features are implemented by the ECS S3 compatible API, so some functionality may not work. However, common file operations (create, read, write and delete) all work well.
 
-## Browser extensions
+## Internet Browser Access
 
-This section takes you through installing and configuring browser extensions that enable you to browse the contents of Cloud Storage using your web browser.
+This section takes you through installing and configuring browser access that enables users to browse the contents of Cloud Storage using their web browser.
 
-Atmos extensions are available for both Chrome and Firefox. Although the extensions included in this article were natively built for the Atmos platform, they also work with ECS using Atmos endpoints and therefore work well with Cloud Storage.
+An EMC created HTML5 based application can be installed to your Atmos bucket, which generates a URL that enables customers to share with users who require access to cloud storage.
```

Comment:
I'd change this to:

An HTML5 application, created by EMC, that you can install in your Atmos bucket to generate a URL that enables you to share the bucket with other users who create access to Cloud Storage.